code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -42,13 +47,23 @@
 		QDEL_NULL(node)
 	. = ..()
 
+/obj/effect/alien/resin/special/pylon/process(delta_time)
+	if(lesser_drone_spawns < lesser_drone_spawn_limit)
+		// One every 10 seconds while on ovi, one every 120-ish seconds while off ovi
+		lesser_drone_spawns = min(lesser_drone_spawns + ((linked_hive.living_xeno_queen?.ovipositor ? 0.1 : 0.008) * delta_time), lesser_drone_spawn_limit)
+
 /obj/effect/alien/resin/special/pylon/attack_alien(mob/living/carbon/xenomorph/M)
 	if(isxeno_builder(M) && M.a_intent == INTENT_HELP && M.hivenumber == linked_hive.hivenumber)
 		do_repair(M) //This handles the delay itself.
 		return XENO_NO_DELAY_ACTION
 	else
 		return ..()
 
+/obj/effect/alien/resin/special/pylon/get_examine_text(mob/user)
+	. = ..()
+
+	. += "Currently holding [SPAN_NOTICE("[Floor(lesser_drone_spawns)]")]/[SPAN_NOTICE("[lesser_drone_spawn_limit]")] lesser drones."
```

Comment:
Have you considered making lesser_drone_spawn_limit instead be the hive's limit on lesser drones? I think it would be better done the otherway so it can convey to the examiner how many lesser drones are possible with the current hive population. But that does mean you likely won't have as much of a backlog of lesser drones available to take. When queen is on ovi the speed currently is rather fast so its not too important to have a buffer available.